### PR TITLE
New geometry: rectangle

### PR DESCRIPTION
**Pull request by michou214**
Add a single/double sided and adjustable rectangle geometry

Parameters :
-  **size x y** where x and y have to be positive
- **doubleSided TRUE/FALSE** to see only one side or both

### CODE REVIEW

**Comment by michou214** [comment]
:+1:

File: projects/objects/geometries/protos/Rectangle.proto [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#VRML_SIM R2019b utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
```
Comment (omichel) [inline projects/objects/geometries/protos/Rectangle.proto]: I believe we should release that under the Apache 2.0 license.

Comment (DavidMansolino) [reply projects/objects/geometries/protos/Rectangle.proto]: This is a more general question, for now, all of our 'home-made' PROTO are using this custom license.
If we want to change this we should probably define (and document) a clear rule and switch them all at the same time.

Comment (omichel) [reply projects/objects/geometries/protos/Rectangle.proto]: OK, let's do it separately.

**Comment by michou214** [comment]
:+1:

**Pull request merged**